common/core/src/commonMain/kotlin/com/trafi/core/ErrorCode.kt

```diff
@@ -0,0 +1,42 @@
+package com.trafi.core
+
+sealed class ErrorCode {
+    enum class Msp(val value: String) {
```

Comment:
Unless you extend the sealed class, they won't be part of the sealed class hierarchy 🙂
```suggestion
    enum class Msp(val value: String): ErrorCode() {
```

If we do this, then perhaps we can reuse `ApiResult.Failure.Error`? And add an `val errorCode: ErrorCode` property.

Reply:
changed ErrorCode to `object`.
as discussed privately, leaving `MspError` and `UserError` for now